### DISCUSSION TITLE
How do I select text, and give it a command?

### DISCUSSION THREAD

**Discussion by Kenljs** [nhaouari/obsidian-textgenerator-plugin#18] [Q&A] [answered]
Hello! This plugin is _amazing,_ and the technology behind it is jaw-dropping, yet i'm having an issue. I want to create study questions from one of my notes. I have a long note which is all of my notes from one of my classes. I want to select the whole text, and then ask textgenerator to "make ten study questions from this document." Yet, I can't figure out how to do this...?

**Comment by Kenljs** [comment]
This is the error code that pops up.

Error: Request failed, status 400
    at new t (app.js:1:699319)
    at Xb (app.js:1:699511)
    at app.js:1:700188
    at app.js:1:235201
    at Object.next (app.js:1:235306)
    at a (app.js:1:234045)
plugin:obsidian-textgenerator-plugin:200 Object
plugin:obsidian-textgenerator-plugin:93 Error: Request failed, status 400
    at new t (app.js:1:699319)
    at Xb (app.js:1:699511)
    at app.js:1:700188
    at app.js:1:235201
    at Object.next (app.js:1:235306)
    at a (app.js:1:234045)

**Comment by nhaouari** [comment, marked as answer]
> Hello! This plugin is _amazing,_ and the technology behind it is jaw-dropping, yet i'm having an issue. I want to create study questions from one of my notes. I have a long note which is all of my notes from one of my classes. I want to select the whole text, and then ask textgenerator to "make ten study questions from this document." Yet, I can't figure out how to do this...?

Hello! 
To select the whole text you need to put the cursor at the end of the document in a new empty line. 
For more information, take a look to  the considered context section.
